README.md

```diff
@@ -1,2 +1,61 @@
-# moz-mlops
-A tooklit for productionizing machine learning models at Mozilla. 
+# Training and Deploying Models in The Cloud
+
+So you're a Mozillian with a machine learning model that you have futzed with locally, or maybe in Google Colab, or possibly even on your own cluster. You’re now ready for a more sophisticated model orchestration setup. 
+
+Excellent! You've found the toolset for this. We use deployed Metaflow flows with Outerbounds to do model orchestration, and we use Weights and Biases for experiment evaluation. The templates in this repository will help you integrate with both.
+
+## But first...you need some stuff.
+
+### Most importantly, you need an account with Outerbounds (do not make this yourself).
+
+An admin from Mozilla’s MLOps team needs to make you a user account in our production Outerbounds instance. You can ask for help with this in the #mlops Slack channel.
+
+**The admins need to know two things:**
+1. If your data requires more restricted access than All-Mozilla. If so, they will handle getting a perimeter created for you, and will get back to you with your Outerbounds login information and perimeter name. 
+1. If you intend to run your flow on your own cluster or your own servers, rather than Outerbounds resources. If yes, they need to set that up. Let them know and they’ll work with you on that.
+
+Once MLOps comes back to you with a shiny new Outerbounds account and a perimeter for your team, you can sign in [here](https://docs.google.com/document/d/12S06Q-9xh6YGkyBpi6iGWGazYDivxl2fT4KaHtbrzV4/edit).
+
+Go to [this page right here](https://ui.desertowl.obp.outerbounds.com/dashboard/configure?location=local). You’ll see some instructions. You have already installed Outerbounds, so you get to skip that one. 
+Look for a dropdown that says “perimeter.” You want to make sure that, if your data has more restricted access than All-Mozilla, you have checked the perimeter name that you received from MLOps. If your data has all-Mozilla access, you can check “default” here.
+
+You should see an instruction to run a command line command that starts with `outerbounds configure`. Copy that whole thing and run it on your command line. What this does, is it makes a `config.json` file located at `~/.metaflowconfig` on your local machine. This is the default configuration for metaflow on your machine now, and unless told otherwise, metaflow flows on your machine will now always run on Outerbounds. 
```

Comment:
That makes sense to me, given that this repo is public and the steps here would be specific to Mozilla. Since I have _not_ set up that Confluence page yet and don't want it to block me on moving this over to the other repo, I'm going to table this suggestion for now and make the change _after_ I get this into the other repo.

Reply:
In the future, I'd appreciate the reason for the change being clearly stated. In this case I could tell, but in other cases, the context for why on changes like this stays in your head and all I see is 'make this change please.'

Reply:
Apologies, I believe my intent was clear because of my top comment [in the original doc](https://docs.google.com/document/d/12S06Q-9xh6YGkyBpi6iGWGazYDivxl2fT4KaHtbrzV4/edit#heading=h.ulsdk1khtl7i). Will make sure to be more explicit next time!